packages/fxa-settings/src/lib/hooks/useFxAStatus/index.tsx

```diff
@@ -61,9 +65,27 @@ export function useSyncEngines(integration: SyncEnginesIntegration) {
             setWebChannelEngines(status.capabilities.engines);
           }
         }
+        // Check if third party auth (passwordless) log in to the browser is supported
+        if (
+          // status.capabilities.keys_optional &&
```

Comment:
This is commented out for testing and will cause some test failures, will be uncommented with next push

Reply:
Noting that commenting out this line mimics enabling the feature, so it's needed for manual testing of the new flows.